### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by snuyanzin**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+4. Commit to your fork using clear commit messages.
+5. Send us a pull request, answering any default questions in the pull request interface.
+6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
+
+GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
+[creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
+
+## Developer Certificate of Origin
+
+JDBC connector for Apache Kafka is an open source product released under the Apache 2.0 license (see either [the Apache site](https://www.apache.org/licenses/LICENSE-2.0) or the [LICENSE.txt file](LICENSE.txt)). The Apache 2.0 license allows you to freely use, modify, distribute, and sell your own products that include Apache 2.0 licensed software.
```
Comment (jlprat) [inline CONTRIBUTING.md:44]: Wrong project name

**Review by jlprat — CHANGES_REQUESTED** [review]
You copy pasted too much, name in the doc is jdbc connector instead of S3

**Review by jlprat — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (snuyanzin) [reply CONTRIBUTING.md]: done

**Review by jlprat — APPROVED** [review]

**Pull request merged**